assets/helpers/stripeCheckout/stripeCheckout.js

```diff
@@ -52,12 +52,20 @@ export const setupStripeCheckout = (
   });
 });
 
-export const openDialogBox = (amount: number, email: string) => {
-  if (stripeHandler) {
+export const openDialogBox = (
+  amount: number,
+  email: string,
+  postDeployTestCallback: Function,
+  isPostDeploymentTestUser: boolean,
```

Comment:
I would wrap the two fields related topostDeploy tests inside an object. Additionally, it would be possible to define a type related to `PostDeploy` data which would contain this two fields.